content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -343,27 +344,29 @@ In the CLI, the access modes are abbreviated to:
-| CephFS               | &#x2713;     | &#x2713;    | &#x2713;     |
-| Cinder               | &#x2713;     | -           | -            |
-| FC                   | &#x2713;     | &#x2713;    | -            |
-| Flexvolume           | &#x2713;     | &#x2713;    | depends on the driver |
-| Flocker              | &#x2713;     | -           | -            |
-| GCEPersistentDisk    | &#x2713;     | &#x2713;    | -            |
-| Glusterfs            | &#x2713;     | &#x2713;    | &#x2713;     |
-| HostPath             | &#x2713;     | -           | -            |
-| iSCSI                | &#x2713;     | &#x2713;    | -            |
-| Quobyte              | &#x2713;     | &#x2713;    | &#x2713;     |
-| NFS                  | &#x2713;     | &#x2713;    | &#x2713;     |
-| RBD                  | &#x2713;     | &#x2713;    | -            |
-| VsphereVolume        | &#x2713;     | -           | - (works when pods are collocated)  |
-| PortworxVolume       | &#x2713;     | -           | &#x2713;     |
-| ScaleIO              | &#x2713;     | &#x2713;    | -            |
-| StorageOS            | &#x2713;     | -           | -            |
+| Volume Plugin        | ReadWriteOnce| ReadOnlyMany          | ReadWriteMany|
+| :---                 |     :---:    |    :---:              |    :---:     |
+| AWSElasticBlockStore | &#x2713;     | -                     | -            |
+| AzureFile            | &#x2713;     | &#x2713;              | &#x2713;     |
+| AzureDisk            | &#x2713;     | -                     | -            |
+| CephFS               | &#x2713;     | &#x2713;              | &#x2713;     |
+| Cinder               | &#x2713;     | -                     | -            |
+| FC                   | &#x2713;     | &#x2713;              | -            |
+| Flexvolume           | &#x2713;     | &#x2713;              | depends on the driver |
+| Flocker              | &#x2713;     | -                     | -            |
+| GCEPersistentDisk    | &#x2713;     | &#x2713;              | -            |
+| Glusterfs            | &#x2713;     | &#x2713;              | &#x2713;     |
+| HostPath             | &#x2713;     | -                     | -            |
+| iSCSI                | &#x2713;     | &#x2713;              | -            |
+| Quobyte              | &#x2713;     | &#x2713;              | &#x2713;     |
+| NFS                  | &#x2713;     | &#x2713;              | &#x2713;     |
+| RBD                  | &#x2713;     | &#x2713;              | -            |
+| VsphereVolume        | &#x2713;     | -                     | - (works when 
+pods are collocated)  |
+| PortworxVolume       | &#x2713;     | -                     | &#x2713;     |
+| ScaleIO              | &#x2713;     | &#x2713;              | -            |
+| StorageOS            | &#x2713;     | -                     | -            |
+|CSI                   | &#x2713;     | depends on the driver |depends on the driver |
```

Comment:
Can we list the Volume Plugin in alphabetical order?

Reply:
thanks,  done